chaoskube/chaoskube.go

```diff
@@ -196,7 +203,10 @@ func (c *Chaoskube) Candidates() ([]v1.Pod, error) {
 	pods = filterByAnnotations(pods, c.Annotations)
 	pods = filterByPhase(pods, v1.PodRunning)
 	pods = filterByMinimumAge(pods, c.MinimumAge, c.Now())
-
+	if c.IncludeRegex != nil {
+		fmt.Println(c.IncludeRegex.String())
```

Comment:
This can be removed as it's already printed in `main.go` and never changes afterwards.